interface/batchcom/batchcom.php

```diff
@@ -24,280 +29,217 @@
 include_once("../../library/acl.inc");
 include_once("batchcom.inc.php");
 
-// gacl control
-$thisauth = acl_check('admin', 'batchcom');
+use OpenEMR\Core\Header;
 
-if (!$thisauth) {
-  echo "<html>\n<body>\n";
-  echo "<p>".xl('You are not authorized for this.','','','</p>')."\n";
-  echo "</body>\n</html>\n";
-  exit();
- }
+if (!acl_check('admin', 'batchcom')) {
+    echo "<html>\n<body>\n";
+    echo "<h1>".xl('You are not authorized for this.','','','</h1>')."\n";
```

Comment:
lets clean up this line a bit and html escape it:
``````
echo "<h1>" . xlt('You are not authorized for this.') . "</h1>";